src/db_queries.js

```diff
@@ -1,11 +1,22 @@
+const joi = require('joi');
+
 dbQueries = {};
 
+const schema = joi.object({
+  name: joi.string().regex(/^[a-zA-Z]{3,40}$/),
+  amount: joi.number().integer()
+})
+
 dbQueries.postTransaction = function(connPool, data, callback) {
-  connPool.query(
-    'INSERT INTO kitty (name, transaction_value) VALUES ($1, $2)',
-    [data.name, data.amount],
-    callback
-  );
+  joi.validate({name: data.name, amount: data.amount}, schema, (err, validated) => {
+    console.log(validated);
+    if (err) throw err;
```

Comment:
I don't think we should be throwing errors here, since it'll crash the whole app. Maybe call the callback the error and return instead.